lib/reports/future_content_change_statistics_report.rb

```diff
@@ -0,0 +1,78 @@
+require "reports/concerns/notification_stats"
+
+class Reports::FutureContentChangeStatisticsReport
+  include Reports::Concerns::NotificationStats
+
+  attr_reader :govuk_path
+
+  def initialize(govuk_path)
+    @govuk_path = govuk_path
+  end
+
+  def call
+    client = GdsApi::JsonClient.new
+    content_json = client.get_json("https://www.gov.uk/api/content#{govuk_path}")
+    content_item = content_json.to_hash
```

Comment:
Just having a cursory glance at this PR and have very little context. Is there a reason you're not using the Content Store GDS API Adapter: https://github.com/alphagov/gds-api-adapters/blob/3e5616df324b90994b961c23b530b98ab8a2641c/lib/gds_api/content_store.rb#L13-L17 ?

For @ChrisBAshton's question about draft stack you could achieve that with a PLEK env var if using the adapter

Reply:
Oh, that's just me being dumb. Because we were talking specifically about the public content api in the other PR, and I didn't join the dots between the Content Store API and the content api just being a publicly exposed subset of that. Okay, let me fiddle further.

Reply:
Updated to use ContentStore API correctly.